tsconfig.json

```diff
@@ -0,0 +1,63 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "module": "es2015",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    //"lib": [ "es6", "dom"],                             /* Specify library files to be included in the compilation. */
+    "allowJs": true,                       /* Allow javascript files to be compiled. */
+    "checkJs": false,                       /* Report errors in .js files. */
+    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
+    // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    "sourceMap": true,                     /* Generates corresponding '.map' file. */
+    // "outFile": "./",                       /* Concatenate and emit output to single file. */
+    "outDir": "./src/js",                        /* Redirect output structure to the directory. */
```

Comment:
care to explain why output to `src`, not something like `dst`?

Reply:
Because webpack will then take the generated *.js and bundle it. At least I think so, it's been a while since I set this template up.

Reply:
Isn't it going to destroy contents in `src`? Or am I misunderstanding the purpose of this file?

Reply:
Well it will only take *.ts and make *.js in the same location. So this should not overwrite anything. Especially since there is no TypeScript in the repo right now.

Reply:
I just don't like build systems which pollute the source directory for building :)